tests/unit_benchmarks/benchmarks.sh

```
@@ -0,0 +1,179 @@
+if [[ $LIST == 1 ]]; then
+    BENCH_ARGS+=" --list"
+fi
+
+E=0
+
+$READIES/bin/sep
+echo "# Running unit benchmarks"
+BENCHS_DIR="$(cd $BINROOT/src/tests/unit_benchmarks; pwd)"
+cd $ROOT/tests/unit_benchmarks
+if [[ -z $BENCH ]]; then
+    for bench in $(find $BENCHS_DIR -type f -executable -name "benchmark_*"); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)